packages/courDeCassation/src/scripts/updateRoute.ts

```diff
@@ -1,30 +1,44 @@
-import yargs from 'yargs';
 import { buildBackend, buildDocumentRepository, logger } from '@label/backend';
 import { parametersHandler } from '../lib/parametersHandler';
-import { documentType } from '@label/core';
 import { extractRoute } from '../connector/mapper/extractors';
+import * as readline from 'readline';
 
 (async () => {
-  const { environment, settings } = await parametersHandler.getParameters();
-  const { status } = parseArgv();
-  const backend = buildBackend(environment, settings);
-
-  backend.runScript(() => updateRoute(status), {
-    shouldLoadDb: true,
+  const prompt = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
   });
+  const { environment, settings } = await parametersHandler.getParameters();
+  prompt.question(
+    'Confirmez vous le lancement de ce script (yes/no)? ',
```

Comment:
```suggestion
    'Voulez vous mettre a jour le circuit de relecture de tous les documents free (yes/no)? ',
```
Sinon bien vu le message de confirmation :ok: